emukit/quadrature/methods/warped_bq_model.py

```diff
@@ -133,3 +134,12 @@ def integrate(self) -> Tuple[float, float]:
         :returns: Estimator of integral and its variance.
         """
         raise NotImplemented
+
+    @staticmethod
+    def _symmetrize(A: np.ndarray) -> np.ndarray:
```

Comment:
This will be used in later PRs, right? It isn't called anywhere in this PR

Reply:
see above